packages/indexer/src/thegraph/oracle.ts

```diff
@@ -1,5 +1,5 @@
 import {GraphCommon} from "./_common";
-import {OracleImportedMessageRoot, QueryLastImportedMessageRoot} from "../types";
+import {OracleImportedMessageRoot, QueryLastImportedMessageRoot} from "../types/graph";
 
 
 export class ThegraphIndexOracle extends GraphCommon {
```

Comment:
I think this is not required?

Reply:
Because I have removed it from ponder indexer, and graph indexer is not exported from index.ts